reference/api/keys.md

```diff
@@ -1,23 +1,24 @@
 # Keys
 
-The `/keys` route allows you to create, manage, and delete API keys. To use these endpoints, you must [set the master key](/learn/configuration/instance_options.md#master-key) and supply it in the header of the request:
-
-```bash
-curl \
-  -H 'Authorization: Bearer MASTER_KEY'
-  …
-```
+The `/keys` route allows you to create, manage, and delete API keys. To use these endpoints, you must first [set the master key](/learn/configuration/instance_options.md#master-key). Once a master key is set, you can access these endpoints by supplying it in the header of the request, or using API keys that have access to the `keys.get`, `keys.create`, `keys.update`, or `keys.delete` actions.
 
 [Learn more about managing keys and their rights](/learn/security/master_api_keys.md).
 
 ## Get all keys
 
 <RouteHighlighter method="GET" route="/keys"/>
 
-List all existing API keys. **Expired keys are included in the response**, but deleted keys are not.
+List the most recently created 20 keys. **Expired keys are included in the response**, but deleted keys are not. Results can be paginated by using the `offset` and `limit` query parameters.
```

Comment:
Same as above. Perhaps it's just a matter of preference?

Googling "the 20 most beautiful" gives me 290.000 results, whereas "the most beautiful 20" gives me 28.000 🤔  (I tried using a sentence a bit more common than "recently created keys").

```suggestion
List the 20 most recently created keys. **Expired keys are included in the response**, but deleted keys are not. Results can be paginated by using the `offset` and `limit` query parameters.
```